src/clarity-angular/popover/dropdown/dropdown.spec.ts

```diff
@@ -19,8 +21,9 @@ export default function(): void {
 
         beforeEach(() => {
             TestBed.configureTestingModule({
-                imports: [ClrDropdownModule, ClrIconModule],
-                declarations: [TestComponent]
+                imports: [ClrDropdownModule, ClrIconModule, ClrConditonalModule],
```

Comment:
So you shouldn't need to import anything other than `ClrDropdownModule` here. That'd be more representative of what we want our consumers to do.